geary-papermc-features/src/main/kotlin/com/mineinabyss/geary/papermc/features/items/resourcepacks/ResourcePackGenerator.kt

```diff
@@ -57,7 +62,15 @@ class ResourcePackGenerator(world: Geary) : Geary by world {
                 .addTo(resourcePack)
         }
 
+        config.jarResources
+            .forEach {
+                resourcePack.unknownFile(it.path, Writable.resource(it.classLoader, it.resource))
+            }
+
+        config.resourcePacks.forEach { resourcePack.merge(it, MergeStrategy.mergeAndFailOnError()) }
```

Comment:
Idofront has built in merge methods u should use as it tries to handle conflicts this might skip